nginx_stage/lib/nginx_stage.rb

```diff
@@ -192,6 +192,21 @@ def self.active_users
     end.compact
   end
 
+  # List of inactive users.
+  # @return [Array<String>] the list of inactive users.
+  def self.inactive_users
```

Comment:
I'm not sure I should be changing the API as it's being used in other places. I guess that's the risk that I'm trading for duplication. I.e., I'd rather have duplication than possible errors/mistakes.

Reply:
I understand your concern. However, I think with sufficient test coverage and carefulness, a change like this shouldn't be an issue. `nginx_stage` is still quite limited in scope compared to the main ondemand application.

It's not a hill I'll die on though, so I'm willing to let this slide. Perhaps one could make a separate issue for it, so that future refactoring can take it into account?

Reply:
Yea the issue is with the 4.0 timeline so I don't have time to write tests or be super careful. If you're happy to defer it, I am too.

Reply:
Sounds good :+1: My other review comment I'm more concerned about, so I'm looking forward to your thoughts on that. Hopefully the suggested commit helps.